.github/workflows/Documenter.yml

```
@@ -1,18 +1,37 @@
 name: Documenter
 on:
-    push:
-        branches: [master]
-        tags: [v*]
-    pull_request:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
 
 jobs:
-    Documenter:
-        name: Documentation
-        runs-on: ubuntu-latest
-        steps:
-        - uses: actions/checkout@v4
-        - uses: julia-actions/julia-buildpkg@v1
-        - uses: julia-actions/julia-docdeploy@v1
-          env:
-            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-            DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+  docs:
+    name: Documentation
+    runs-on: ubuntu-latest
+    permissions:
+      # needed to allow julia-actions/cache to proactively delete old caches that it has created
+      actions: write 
+      contents: write
+    steps:
```

AbstractFFTs publishes a preview (for PRs that are not from forks) (https://github.com/JuliaMath/AbstractFFTs.jl/blob/2ae97a7c06b0c7b43a5ee7d1298d882473a575aa/docs/make.jl#L16), so it would be convenient to add back
```suggestion
      # needed to allow the workflow to create a status with a link to the docs preview
      statuses: write
    steps:
```